### PR TITLE
tweak: add a 'understood' button for the deprecated warning

### PR DESCRIPTION
**Pull request by lze3**
it's a bit tedious having that prompt pop up each time we use it. yes, it's deprecated, but those who use it probably know why they're using it

### CODE REVIEW

**Comment by lze3** [comment]
Could perhaps return if result != Yes?

File: DS4Windows/DS4Forms/MainWindow.xaml.cs [hunk 1]
```diff
@@ -1288,7 +1288,20 @@ private void NewProfBtn_Click(object sender, RoutedEventArgs e)
...
+                MessageBoxResult result = MessageBox.Show
+                (
+                    "This feature is deprecated and no longer supported. Exclusive mode usage is provided mearly as a legacy feature. Do NOT ask for help for this feature, you will not recieve any.\n\nDo you understand?",
+                    "Deprecation Warning",
+                    MessageBoxButton.YesNo,
+                    MessageBoxImage.Warning,
+                    MessageBoxResult.None
+                );
+
+                Properties.Settings.Default.ContCk_Understood = result == MessageBoxResult.Yes;
```
Comment (lze3) [inline DS4Windows/DS4Forms/MainWindow.xaml.cs:1302]: Could maybe return if the user did not understand, we don't want them using a feature, expecting support if they don't know what the feature is or they don't understand the deprecation warning.

```suggestion
                if (result != MessageBoxResult.Yes) return;

                Properties.Settings.Default.ContCk_Understood = true;
```

File: DS4Windows/DS4Forms/MainWindow.xaml.cs [hunk 2]
```diff
@@ -1288,7 +1288,20 @@ private void NewProfBtn_Click(object sender, RoutedEventArgs e)
         // Ex Mode Re-Enable
         private async void HideDS4ContCk_Click(object sender, RoutedEventArgs e)
         {
-            if (DS4Windows.Global.UseExclusiveMode == true) { MessageBox.Show("This feature is depreciated and no longer supported. Exclusive mode usage is provided mearly as a legacy feature. Do NOT ask for help for this feature, you will not recieve any.", "Feature no longer supported"); }
+            if (DS4Windows.Global.UseExclusiveMode == true && !Properties.Settings.Default.ContCk_Understood)
+            {
+                MessageBoxResult result = MessageBox.Show
```
[the remark this answers is not in the document]
Comment (lze3) [reply DS4Windows/DS4Forms/MainWindow.xaml.cs:1293]: nice email address

File: DS4Windows/Properties/Settings.Designer.cs [hunk 3]
```diff
@@ -22,5 +22,17 @@ internal sealed partial class Settings : global::System.Configuration.Applicatio
                 return defaultInstance;
             }
         }
+        
+        [global::System.Configuration.UserScopedSettingAttribute()]
```
[the remark this answers is not in the document]
Comment (lze3) [reply DS4Windows/Properties/Settings.Designer.cs:26]: @Ryochan7 sorry, but can you block the user above from commenting? It’s more or less just spam.

**Comment by Ryochan7** [comment]
Thought about this off and on. After some recent issues, I am going with my original decision that a user **should** be nagged about this feature each time it is enabled. I have been contemplating just stripping the broken feature out again.

As such, these changes will not be merged in.

**Pull request closed without merging**